2023/YoctoDevDay/fido-device-onboarding.md

```diff
@@ -0,0 +1,24 @@
+Title:
+FIDO Device Onboarding: Late-binding Provisioning & Tales from the Trenches of
+Bleeding Edge Tech
+
+Abstract:
+This talk explores the role of FIDO Device Onboarding [1] in strengthening the device provisioning
+process, emphasizing both its security and extensibility. I'll also share my
+personal experience and challenges faced while implementing the
+fido-device-onboard-rs [2] project in the YOCTO environment, offering a practical
+perspective on working with this promising technology
+
+Description:
+In this presentation, we'll delve into how FDO [1] offers innovative solutions
+to prevalent challenges in embedded device onboarding. We'll explore existing
+implementations of this protocol and their compatibility within the Yocto
+Environment. A closer look will be given to the fido-device-onboard-rs [2]
+project by Fedora IoT, which I employed in a recent project. I'll share
+firsthand challenges faced in integrating it with Yocto, especially the
+nuances of cross-compiling Rust and the solutions available in OpenEmbedded
+layers. I'll conclude the presentation by showcasing a live demonstration of
+the protocol in action on a Raspberry Pi platform.
```

Comment:
I like it. 

@TomaszAIR @artur-rs I think by default, we should market approved talks to communities that may be interested and ask them (after the video is published) if there is a place to link the talk somewhere in their project. Also, if we submit any issues regarding problems we faced, which we will present how to overcome, we should like a presentation on that issues. Those are post-presentation actions.

Reply:
> we should market approved talks to communities that may be interested and ask them (after the video is published) if there is a place to link the talk somewhere in their project

@pietrushnic Sure, we will do so. Also we need to find out how and where we could help with Yocto support for that project because now it has like zero support IIRC